src/main/java/org/rust/lang/doc/psi/RustDocElementTypes.java

```diff
@@ -0,0 +1,13 @@
+package org.rust.lang.doc.psi;
+
+public interface RustDocElementTypes {
+    RustDocTokenType DOC_TEXT = new RustDocTokenType("<DOC_TEXT>");
+    RustDocTokenType DOC_DECO = new RustDocTokenType("<DOC_DECO>");
```

Comment:
I just don't see any reason to introduct separate entity, though i may be wrong.